.github/workflows/installer.yml

```diff
@@ -81,6 +81,25 @@ jobs:
       # tar and gzip are needed by the actions/checkout below.
       run: zypper --non-interactive install tar gzip ${{ matrix.deps }}
       if: contains(matrix.image, 'opensuse')
+      # Parrot recently (~2025-01-11) rotated their GPG keys for package signing
+      # but have not (as of 2025-01-24) updated their LTS image to include these
+      # new GPG keys. This image does not come with CURL / wget / a sane way of
+      # fetching the .deb file that their blog post around the key rotation points
+      # to (see https://parrotsec.org/blog/2025-01-11-parrot-gpg-keys/) so instead
+      # use a combination of openssl s_clients and sed to download the .deb and strip
+      # the header information from the resulting file so that it is a a valid .deb
+      # that we can install.
+    - name: install new GPG key (parrotsec/core:lts-amd64)
+      run: |
+        printf "GET /parrot/pool/main/p/parrot-archive-keyring/parrot-archive-keyring_2024.12_all.deb HTTP/1.1\r\nHost: www.deb.parrot.sh\r\nConnection: close\r\n\r\n" | openssl s_client -connect www.deb.parrot.sh:443 -quiet | sed  '/<arch>/,$!d' > parrot-archive-keyring_2024.12_all.deb 
+        SHA_VAL=$(sha256sum parrot-archive-keyring_2024.12_all.deb)
+        if "$SHA_VAL" != "188c74cf726fad499ec416dc6b7dcd9718800aa7178290c10ef7040aea9b5272"
+        then
+          echo "Unexpected SHA for deb"
+          exit 1
+        fi
+        dpkg -i parrot-archive-keyring_2024.12_all.deb
```

Comment:
So the goal of this Github action check is to verify that our installer script works. If we're comfortable supporting this code, we maybe we should do it in the installer script instead, fixing this for other users and not just for our test? If we're not, we could just remove `parrotsec/core:lts-amd64` from the list of covered docker images.

Reply:
Good callouts! I would honestly prefer removing `parrotsec/core:lts-amd64` if we're alright with dropping testing for that image since this code is pretty inscrutable on first read.

Otherwise we could add this into the installer as a special case for parrot, and add this method of fetching as an alternate for curl / wget in general as a final fallback with some refactoring.

Reply:
I would be supportive of dropping it, especially since we have another flavour of `parrotsec` in the list.

Reply:
Cool! Will remove this then